### PR TITLE
[APT-2073] Enable searching KB discussions

### PR DESCRIPTION
**Pull request by eak12913**
This PR adds a bunch of "discussions" pages that are generated by the docs-sourcer and mirror the official Gruntwork Knowledge Base discussions. We also update our search configuration to index these new pages, in effect, making it possible to search the KB discussions from the docs site.

Related docs-sourcer changes: https://github.com/gruntwork-io/docs-sourcer/pull/88

### CODE REVIEW

File: docsearch-local.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "sitemap_alternate_links": true,
   "selectors": {
     "lvl0": {
-      "selector": "(//nav[contains(@class,'menu')]/ul/li/div/a[contains(@class, 'active')]/text() | //nav[contains(@class, 'navbar')]//a[contains(@class, 'navbar__link--active')]/text())[last()]",
+      "selector": "(//nav[contains(@class,'menu')]/ul/li/div/a[contains(@class, 'active')]/text() | //nav[contains(@class, 'navbar')]//a[contains(@class, 'navbar__link--active')]/text())[last()] | //span[contains(@class, 'title')]",
```
Comment (ebeneliason) [inline docsearch-local.json:7]: Maybe call this something more explicit like `search-category`?

File: src/components/GitHub.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from "react"
+import Admonition from "@theme/Admonition"
+
+type GitHubProps = {
+  json: any
+}
```
Comment (ebeneliason) [inline src/components/GitHub.tsx:6]: Add a type

File: src/components/GitHub.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+export const PWithData: React.FunctionComponent<GitHubProps> = ({ json }) => {
+  return <div dangerouslySetInnerHTML={{ __html: json }} />
```
Comment (ebeneliason) [inline src/components/GitHub.tsx:9]: Try taking raw HTML and render it into the page directly

File: src/components/GitHub.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      <div style={{marginTop: "2.5rem"}}>
+        <Admonition type="tip" title="Answer">
+          <PWithData json={answerBody} />
+        </Admonition>
+      </div>
+
+
+
+      <p style={visuallyHidden}>{json.body}</p>
+      <p style={visuallyHidden}>{json.answer?.body}</p>
```
Comment (ebeneliason) [inline src/components/GitHub.tsx:56]: Add a comment here

File: src/components/GitHub.tsx (hunk 3 above)
Comment (eak12913) [reply src/components/GitHub.tsx:9]: I have tried it and it's just not possible to make this work. It doesn't make it past the Docusaurus build if the HTML is injected directly into the MDX files.

**Review by ebeneliason — APPROVED** [review]
Updates look great.

**Pull request merged**